tests/test_diff.py

```diff
@@ -14,9 +14,10 @@ def test_diff():
         engine,
         'SELECT * FROM sensor_readings WHERE temperature > 50',
         'SELECT * FROM sensor_readings WHERE temperature <= 50',
-        (Column('created_at'), Column('sensor_id'), Column('voltage'),
-         Column('humidity')),
-        .05,
+        {Column('created_at'), Column('sensor_id'), Column('voltage'),
+         Column('humidity')},
+        {Column('voltage')},
```

Comment:
Can we make it such that a bucket arises as a reason, or should we leave well enough alone so it's closer to the example paper's result?

Reply:
The dataset is too small to see an effect, so I created #25 to augment the test suite in a separate PR